edgelet/edgelet-docker/src/runtime.rs

```diff
@@ -631,6 +631,9 @@ impl ModuleRuntime for DockerModuleRuntime {
                 })
             });
 
+        #[cfg(not(any(windows, target_os = "linux")))]
+        let uptime: u64 = 0;
+
         #[cfg(target_os = "linux")]
```

Comment:
will this work if you simply change to this?
```suggestion
        #[cfg(target_os = "unix")]
```

Reply:
target_os = "unix" doesn't build. There's actually a "macos" target_os. But, libc doesn't have sysinfo for "macos" platform.

Reply:
`#[cfg(unix)]` its not a target_os, but family. but it won't work anyways due to sysinfo limitations...